Training/PythonWebServer/webserver.py

```diff
@@ -2,20 +2,45 @@
 
 from http.server import SimpleHTTPRequestHandler, HTTPServer
 import sys
+import psutil
+from datetime import datetime
+import pytz
 
 class SoarWebServer(SimpleHTTPRequestHandler):
 
     def do_POST(self):
-        self.wfile.write("Send your POST response here!")
+        #Send your POST response here!
+        s0 = str(datetime.now(tz=pytz.timezone('America/Edmonton')).strftime('%H:%M:%S'));
+        s1 = str(psutil.cpu_count());
+        s2 = str(psutil.cpu_percent(interval=1));
+       	s3 = str(round(psutil.virtual_memory().total/(2**30), 2));
+        s4 = str(round(psutil.virtual_memory().used/(2**30), 2));
```

Comment:
It's important to pick descriptive variable names, `s4` doesn't really tell a reader what kind of data is in the variable. For example, the `s4` variable would be better named `virtual_memory_used`. The other variables in this function could use some more descriptive names too :)